petab/visualize/plotting_config.py

```diff
@@ -72,42 +72,41 @@ def plot_lowlevel(vis_spec: pd.DataFrame,
         # add xOffset
         conditions = conditions + vis_spec.xOffset[i_visu_spec]
 
+        # TODO sort mean and sd/sem by x values (as for simulatedData below)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [TODO sort mean and sd/sem by x values (as for simulatedData below)](https://app.codacy.com/manual/dweindl/PEtab/pullRequest?prid=4827411)